src/EditorFeatures/Next/project.json

```diff
@@ -1,22 +1,23 @@
 {
   "dependencies": {
-    "Microsoft.VisualStudio.CoreUtility": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Editor": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Imaging": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.ImageCatalog": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Language.Intellisense": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Text.Data": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Text.Logic": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Text.UI": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Utilities": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Language.NavigateTo.Interfaces": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Utilities": "15.0.25726-Preview5",
-    "Microsoft.VisualStudio.Validation": "15.0.55-pre",
-    "Microsoft.VisualStudio.Threading": "15.0.109-pre",
+    "Microsoft.VisualStudio.CoreUtility": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Editor": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Imaging": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
+    "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": "14.3.25407",
+    "Microsoft.VisualStudio.ImageCatalog": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.Intellisense": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.Data": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.Logic": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.UI": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Utilities": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.NavigateTo.Interfaces": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Utilities": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Validation": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Threading": "15.0.26014-alpha",
     "System.Collections.Immutable": "1.3.1",
-    "System.Threading.Tasks.Dataflow": "4.5.24",
+    "Microsoft.Tpl.Dataflow": "4.5.24",
```

Comment:
There are official VS packages that we depend on which reference Microsoft.Tpl.Dataflow.  Trying to mix System.Threading.Tasks.Dataflow in with this dependency causes resolve / double write issues in the build.  Hence I had to stop fighting this and move to Microsoft.Tpl.Dataflow. 

I've updated the tracking issue on CoreFX noting the additional pain this is causing.  Once this is available I can contact the VS owners and ask them to adjust their dependencies.  Until then we are stuck. 

Welcome to my nightmare.